### PR TITLE
CONSOLE-2775: Consolidate i18n namespaces n-r

### PR DESCRIPTION
**Pull request by zherman0**
Consolidate i18n namespaces n-r

### CODE REVIEW

File: frontend/packages/patternfly/locales/en/notification-drawer.json [hunk 1]
```diff
@@ -0,0 +1,11 @@
+{
+  "Notifications": "Notifications",
```
Comment (dtaylor113) [inline frontend/packages/patternfly/locales/en/notification-drawer.json:2]: Hi @zherman0, what is creating this file in `packages/patternfly`?  It looks like you removed all the `~notification-drawer` i18n namespaces.  How is it different from the deleted `frontend/public/locales/en/notification-drawer.json`?

**Comment by zherman0** [comment]
/assign @yapei @ahardin-rh @sferich888
for approvals

File: frontend/packages/patternfly/locales/en/notification-drawer.json (hunk 1 above)
Comment (zherman0) [reply frontend/packages/patternfly/locales/en/notification-drawer.json:2]: @dtaylor113  -  package specific translations need to be in the package.

**Review by dtaylor113 — APPROVED** [review]
/lgtm

**Comment by spadgett** [comment]
/label px-approved
We've received approval from Eric for this PR.

**Review by spadgett — APPROVED** [review]
/approve

LGTM, but will need rebase

**Review by spadgett — APPROVED** [review]
/lgtm

**Comment by zherman0** [comment]
/retest

couchbase error timeout looking for button which seems to be a reoccurring flake

**Review by spadgett — APPROVED** [review]
/lgtm

**Review by spadgett — APPROVED** [review]
/lgtm

**Pull request merged**